device/mellanox/x86_64-nvidia_sn4280_simx-r0/ACS-SN4280/create_only_config_db_buffers.json

```diff
@@ -0,0 +1,7 @@
+{
+    "DEVICE_METADATA": {
```

Comment:
As i understand this is used only for non-MSFT SKU's.

`https://github.com/sonic-net/sonic-buildimage/pull/16233`. Do we need this here?

Reply:
okay, sure